### PR TITLE
Restaurant Page project is finished

### PR DESCRIPTION
**Pull request by elmejdki**
I kindly request to review my project :smiley: this is the JavaScript Restaurant page project, where have followed the next steps:
:point_down: :point_down: :point_down: :point_down:
- I have used ES6 modules,
- Set up WebPack properly,
- Developed the Four pages using vanilla JavaScript,
- And deployed to GitHub pages as suggested by the OdinProject.

### CODE REVIEW

**Comment by sumancrest0001** [comment]
**Status: Changes required :gear:
Hello @elmejdki,** Your file structure and code look good but I had a problem while running it. There are small changes required to get approved. Let's consider the following points.

[Required] The built js file should be main.js, not bundle.js. [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/dist_requirement.png)

[Required] I got the following error while running the project. Let me know whether it is running correctly on your machine or not.
Screenshot from 2020-10-03 09-33-04

[Required] Please add installation and running steps in the readme file to run the project.

[Optional] Live link is not added correctly in the readme file. Please add if you have.

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
**Make the required changes and send it for another review. If you have any query feel free to message me on slack. Suman Shrestha
Happy coding !!!**

**Comment by elmejdki** [comment]
## Hi, @sumancrest0001 thank you very much for your code review.

- I have changed the bundle file name to main.js

- eventually, I don't have that error at all, maybe you forgot to run npm install that's why you are getting that error.

- I have added the installation and running steps to the readme file.

- for the live link I had a typo before now it's fixed, you can check it.

### Best Regards.

File: README.md [hunk 1]
```diff
@@ -1,57 +1,71 @@
...
+Then you should run: `npm install`
 
-### Install
+And afterwards you supposed to run: `npm run server`, to run the project in your local machine.
 
-### Usage
+Congrats the project is fully working.
 
-### Run tests
+_You may have some issues with the `image-webpack-loader`  but that depend on your operating system each operating system has his own way to solve if you goggled it just for some couple minutes I'm sure you will find the solution, if not you can contact me any time you need I will be happy to help, my contact info are below_
```
Comment (addod19) [inline README.md:40]: [OPTIONAL]
- I think this part of the text should rather be in the PR. Congratulations :fireworks:

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @elmejdki, congratulations :100: Having fixed all the comments made by the previous TSE :+1: I HEREBY APPROVE THE PROJECT. Congratulations go ahead and merge into development :1st_place_medal: 

_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

File: README.md (hunk 1 above)
Comment (elmejdki) [reply README.md:40]: _Hi, I have already thought of that, to be honest, but not every user of the app will look in the PR's most of the users read the readme only._

**Pull request merged**